runtime/src/bank.rs

```diff
@@ -4399,6 +4445,15 @@ impl Bank {
             self.rewrite_stakes();
         }
 
+        if new_feature_activations.contains(&feature_set::simpler_capitalization::id()) {
+            let existing_sysvar_account_count = 9;
+            let existing_native_program_account_count = 7;
```

Comment:
if I killed the cluster, this would mean I even fail the primary math class, I guess... ;)